src/metabase/api/app.clj

```diff
@@ -288,11 +290,13 @@
                                       {:status-code 400})))
                   pk-param (first pks)
                   pk-field-slug (:id pk-param)
-                  model-action (get model-id->model-actions model-id)
+                  actions (get model-id->actions model-id)
                   ident-type "model"]
             page-type ["list" "detail"]
-            :let [selector (if (= page-type "detail") :requires_pk (complement :requires_pk))
-                  actions (->> model-action
+            :let [selector (if (= page-type "detail")
+                             requires-pk
+                             (complement requires-pk))
+                  actions (->> actions
                                (filter selector)
                                (map action-mapper)
                                (sort-by (comp sorter first)))]]
```

Comment:
Is `sorter` going to get a kind here? `action-mapper` seems to return an action-id as the first element and kind doesn't seem to be in the tuple at all.